src/Types/PreparedPerson.ts

```diff
@@ -0,0 +1,6 @@
+import { Person } from './Person';
+
+export interface PreparedPerson extends Person{
+  father: Person | undefined,
+  mother: Person | undefined,
```

Comment:
use `null` instead of `undefined` (`null` is more about objects than `undefined`)
```suggestion
  father: Person | null,
  mother: Person | null,
```